rbc_base/MotorAdaptiveState[3] adaptive_states